test/e2e/alcotest/failing/check_located.expected

```diff
@@ -0,0 +1,70 @@
+Testing `test/e2e/alcotest/failing/check_located.ml'.
+This run has ID `<uuid>'.
+
+ASSERT Expected failure
+  [FAIL]        check                 0   here.
+ASSERT Expected failure
+  [FAIL]        check                 1   pos.
+ASSERT Expected failure
+  [FAIL]        check_raises          0   here.
+ASSERT Expected failure
+  [FAIL]        check_raises          1   pos.
+ASSERT Expected failure
+  [FAIL]        fail                  0   here.
+ASSERT Expected failure
+  [FAIL]        fail                  1   pos.
+
+┌──────────────────────────────────────────────────────────────────────────────┐
+│ [FAIL]        check                 0   here.                                │
```

Comment:
I see that the error messages are descriptive (which is great!) but the `1 pos.` confuses me a bit. What are the `1 pos.` or `0 here` supposed to indicate?

Reply:
I agree that looks confusing here; it's actually a general feature of Alcotest output: tests are printed with their index + name, so for a more normal set of tests it might look like this:

```
  [OK]          FS                  0   Basic operations on contents.
  [OK]          FS                  1   Basic operations on nodes.
  [OK]          FS                  2   Basic operations on commits.
  [OK]          FS                  3   Basic operations on branches.
  [OK]          FS                  4   Hash operations on trees.
  [OK]          FS                  5   Basic merge operations.
  [OK]          FS                  6   Basic operations on slices.
  [OK]          FS                  7   Test merges on tree updates.
  [OK]          FS                  8   Tree caches and hashconsing.
  [OK]          FS                  9   Complex histories.
  [OK]          FS                 10   Empty stores.
  [OK]          FS                 11   Private node manipulation.
```

Reply:
Ooh okay, that makes sense. Thanks! LGTM! :)